README.md

```diff
@@ -99,6 +100,9 @@ sudo apt-get install \
     python2.7-dev
 
 cabal install happy
+
+cd build # or wherever you ran CMake before
+cmake .. -DBOND_ENABLE_COMM=TRUE -DBOND_ENABLE_GRPC=TRUE -DgRPC_ZLIB_PROVIDER=package
```

Comment:
well this is only the case if it is previously installed, since in windows it would work without